css/home.css

```diff
@@ -1,4 +1,39 @@
 /* Home page specific styles */
+@media only screen and (min-width: 960px) {
+.footer {
+  padding: 10px;
+  padding-bottom: 4px;
+  background-color: #003c8f;
+  width: 1366px
+  heigth: 270px
+  margin: 0 auto;
+  overflow: hidden;
+  position: relative;
+}
+
+.footer__text {
+  word-break: break-word;
+  text-align: center;
+  font-size: 16px;
+  color:white;
+  font-family: Roboto;
+}
+
+.footer__icons:nth-last-child(1) {
+  margin: 0;
+  display: inline-block;
+  padding:34px;
+}
+
+.footer__hover{
```

Comment:
this block could probably work best if you rename `footer__hover` to `footer__footnote` as `footer__hover` is just confusing.

then change the code from lines 28,29 and 30 to be something like

```
footer__footnote:hover {
  color: white;
}
```
that on the strictly coding side, but, also isn't this line of text always white?
if so this block of code is just useless.

i know this is a long comment, please write to me if you need any explanation.

Reply:
hey , i have a issue with de link is not show in white underline , i saw in blue and losses with color blue in the background because of that i have to do that css rule.